app/presenters/email_alert_presenter.rb

```diff
@@ -0,0 +1,56 @@
+class EmailAlertPresenter
+  attr_reader :document
+
+  def initialize(document)
+    @document = document
+  end
+
+  def to_json
+    {
+      subject: document.title + " updated",
+      body: body,
+      links: { topics: [document.content_id] },
+      document_type: document.publishing_api_document_type,
+    }.merge(extra_options)
+  end
+
+  def body
+    view_renderer.render(
+      template: "email_alerts/publication",
+      formats: ["html"],
+      locals:   {
+        document_title: document.title,
+        document_summary: document.summary,
+        document_url: [Plek.current.website_root, document.base_path].join(""),
```

Comment:
You should probably use `File.join` here. There is also `URI.join` but that's a bit temperamental. `File.join` would handle any missing slashes.